src/components/top-bar/top-bar.tsx

```diff
@@ -22,8 +26,19 @@ const COMPONENT_NAME = 'TopBar';
  * @example <TopBar>...</TopBar>
  * @extends {Component}
  */
-@connect(mapStateToProps)
+@withPlayer
+@withEventManager
+@connect(mapStateToProps, bindActions(actions))
 class TopBar extends Component<any, any> {
+  topBarRef: RefObject<HTMLDivElement> = createRef<HTMLDivElement>();
+
+  componentDidMount() {
+    const {player, eventManager} = this.props;
+    eventManager.listen(player, player.Event.UI.USER_OPEN_DROPDOWN, () => {
+      this.props.updateTopBarClientRect(this.topBarRef.current?.getBoundingClientRect());
```

Comment:
now that you've checked that this works - can we listen to an earlier event or to an earlier state change in order to know that we can now compute top bar size ?
is the height on componentDidMount already the full height or not yet ?

Reply:
I already checked if we have this in componentDidMount but no we don't have the sizes yet